tests/package-title-dir-rtl-001/OPS/content_001.xhtml

```diff
@@ -0,0 +1,18 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+
+<head>
+    <title>Package Title Direction RTL 001</title>
```

Comment:
Nit: I'm not really setting the XHTML <title> element (do RSes ever show it?), but if we're going to, I think it should be similar to the test ID, rather than 001. That is, something something dir="rtl".

Reply:
Ah, I see that you have not set the title in cmt tests, just left it 'Title'... However, the fact is that test providers (at least I) also look at the content in a traditional browser to check whether it is all right, and therefore the title may be helpful (you are right that RS-s ignore it).

Using the title as appear in the package is a good idea. That makes it clean.

Reply:
Done it in https://github.com/w3c/epub-tests/pull/31/commits/dc9f0d396bd4d814812b3d707225bef4f0f59357